sdk/compute/Microsoft.Azure.Management.Compute/tests/ScenarioTests/VMTestBase.cs

```diff
@@ -1145,13 +1145,13 @@ protected DedicatedHost CreateDedicatedHost(string rgName, string dedicatedHostG
             {
                 if(vm.StorageProfile.DataDisks.Any(dd => dd.ManagedDisk != null && dd.ManagedDisk.StorageAccountType == StorageAccountTypes.UltraSSDLRS))
                 {
-                    Assert.NotNull(vm.AdditionalCapabilities);
-                    Assert.NotNull(vm.AdditionalCapabilities.UltraSSDEnabled);
-                    Assert.True(vm.AdditionalCapabilities.UltraSSDEnabled.Value);
+                    Assert.NotNull(vmOut.AdditionalCapabilities);
```

Comment:
why is this changed?

Reply:
As discussed offline, just to have a better validation in place since it should read from vm response. Reverting it for now as it's not in the scope of this PR